letsdrill.rb

```diff
@@ -1,12 +1,38 @@
-def get_letter_grade(integer)
+def get_letter_grade(grade)
  
   #Put your code here!
+  case grade
+  when 0..64
+  	return "F"
+  when 65..69
+  	return "D"
+  when 70..79
+  	return "C"
+  when 80..89
+  	return "B"
+  when 90..100
+  	return "A"
+  else
+  	return -1
+  end
 
 end
 
 def shortest_string(array)
+  if array.empty?
```

Comment:
nice check for empty here. you'll usually find this written as a oneliner `return nil if array.empty?`. in most cases, however there are better ways to return nil than explicitly. for example this entire method could also be written as...
```ruby
def shortest_string(array)
  array.sort_by(&:length).first
end
```